.github/workflows/remove-from-stage.yml

```diff
@@ -0,0 +1,25 @@
+name: Remove from Staging
+
+on: 
+  delete:
+    branches:
+      - '*'
+      - '!main'
+
+jobs:
+  remove:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Extract branch name
+      shell: bash
+      run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      id: extract_branch
+    - name: Remove from s3
+      uses: vitorsgomes/s3-rm-action@master
```

Comment:
Please pin 3rd party actions to a commit hash. Otherwise a compromise of this repo could gain control of our AWS access keys.